client/client-multi/client-objective-c/Kaa/channel/failover/DefaultFailoverManager.m

```diff
@@ -16,15 +16,14 @@
 
 #import "DefaultFailoverManager.h"
 #import "KaaLogging.h"
+#import "DefaultFailoverStrategy.h"
+#import "KaaExceptions.h"
 
-#define TAG @"DefaultFailoverManager >>>"
+static const NSString *logTag = @"DefaultFailoverManager >>>";
```

Comment:
const here doesn't do anything really, NSString is immutable by itself. Maybe you wanted to say "static NSString *const"  as in constant pointer?